### PR TITLE
Integrate sentry

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Integrates Sentry, to track down errors in the client side.

I tried to force one error and I can see it in app:

image

# To Test

1.  You can try to login in Sentry
2. Force an error

I just added this to the CowGame page to try out locally (and added the `REACT_APP_SENTRY_DSN` to my local env). I know this steps won't be easy for everyone to follow, I guess you will need to trust me that it works, or ask for help :)  
```js
if (0 === 0) {
    throw new Error('CowSwap is loosing some milk!')
  }
```

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ env:
   APP_ID: 1
   REPO_NAME_SLUG: gpswapui
   PR_NUMBER: ${{ github.event.number }}
+  REACT_APP_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```
Comment (alfetopito) [inline .github/workflows/ci.yml:16]: Have you added that to repo settings already?

File: src/custom/pages/App/index.tsx [hunk 2]
```diff
@@ -10,6 +10,23 @@ import TermsAndConditions from 'pages/TermsAndConditions'
...
+
+if (SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.REACT_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    // We recommend adjusting this value in production
+    tracesSampleRate: SENTRY_TRACES_SAMPLE_RATE ? Number(SENTRY_TRACES_SAMPLE_RATE) : 1.0,
```
Comment (alfetopito) [inline src/custom/pages/App/index.tsx:27]: Is there a rate limit or something?

**Review by alfetopito — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/ci.yml:16]: yep! 👌

File: src/custom/pages/App/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/custom/pages/App/index.tsx:27]: They make u pay if u pass some amount of errors. For now i left this that is the default, we just need to code with no bugs so there's no a lot of notifications :P 

Well see how many we get, and adjust

**Pull request merged**